### PR TITLE
Add `How I SBT` by `Higher-Kinded Type` to `learn` page

### PR DESCRIPTION
**Pull request by Friendseeker**
c.c. https://discord.com/channels/632150470000902164/922600050989875282/1308963695648571392

FYI @vivekragunathan I, just like Seth Tisue, found your series of blogposts on `sbt` to be really well written! Especially the [2nd entry](https://blog.rhetoricalmusings.com/posts/how-i-sbt2) clarified essential sbt concept (Settings & Tasks) in a very succinct & clear manner.

### CODE REVIEW

File: src/pages/learn.mdx [hunk 1]
```diff
@@ -60,6 +60,7 @@ Blog posts
 - [sbt - A task engine](https://jazzy.id.au/2015/03/03/sbt-task-engine.html), 2015, James Roper
 - [sbt - A declarative DSL](https://jazzy.id.au/2015/03/04/sbt-declarative-dsl.html), 2015, James Roper
 - [testing sbt plugins](https://eed3si9n.com/testing-sbt-plugins), 2011, Eugene Yokota
+- [How I SBT](https://blog.rhetoricalmusings.com/posts/how-i-sbt1/), 2024, Vivek Raghunathan
```
Comment (vivekragunathan) [inline src/pages/learn.mdx]: typo: Vivek Ragunathan (no h)

***

**`[not-an-action-item]`** Not an issue, I prefer the name `Higher-Kinded Type`, as that is a bit widely known 😄

**Review by vivekragunathan — APPROVED** [review]

File: src/pages/learn.mdx (hunk 1 above)
Comment (Friendseeker) [reply src/pages/learn.mdx]: @eed3si9n I saw all other entries in `learn` page use legal name to specify author. Is this a convention or is it okay to use the name `Higher-Kinded Type`?

or maybe `Vivek Raghunathan (Higher-Kinded Type)`?

**Review by vivekragunathan — APPROVED** [review]

File: src/pages/learn.mdx (hunk 1 above)
Comment (eed3si9n) [reply src/pages/learn.mdx]: We should respect the chosen / preferred name of each authors. I have zero mandate for legal name.

@vivekragunathan It's up to you. We can list you as 
1. Higher-Kinded Type
2. Vivek Ragunathan (Higher-Kinded Type)
3. Higher-Kinded Type (Vivek Ragunathan)

Comment (vivekragunathan) [reply src/pages/learn.mdx]: I like option #3.

**Review by vivekragunathan — APPROVED** [review]

**Review by eed3si9n — APPROVED** [review]
Thanks!

**Pull request merged**